spec/features/user/retired_item_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+feature 'when an item is retired' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.